### DISCUSSION TITLE
Docs for Native Fetch

### DISCUSSION THREAD

**Discussion by Robbie-Microsoft** [nodejs/help#4650] [Q&A] [answered]
I see that NodeJS's native fetch is stable in versions 21+. Are there docs that explain how to use this API? I'm trying to figure out if there's proxy support.

**Comment by atlowChemi** [comment, marked as answer]
You could checkout the docs of `undici`, which is the source of node's `fetch`: https://undici.nodejs.org/#/?id=undicifetchinput-init-promise
